static/src/styles/core/color.styl

```diff
@@ -1,12 +1,15 @@
 $Color__light = #F2F2F2;
+$Color__medium = #878787;
```

Comment:
There isn't a great answer here, and we don't need to break our brains figuring it out so `mediumGray` sounds good to me (note the US spelling :cry:)